nuclia/sdk/kb.py

```diff
@@ -241,14 +244,28 @@ def copy(
                 else:
                     files_to_upload.append({"id": file_id, "data": file.value})
         destination_kb = get_client(destination)
-        uuid = self.resource.create(ndb=destination_kb, **data)
-        with tempfile.TemporaryDirectory() as tmpdirname:
-            for file_data in files_to_upload:
-                file_path = os.path.join(tmpdirname, file_data["data"].file.filename)
-                self.resource.download_file(
-                    rid=res.id, file_id=file_data["id"], output=file_path, **kwargs
-                )
-                self.upload.file(path=file_path, ndb=destination_kb, rid=uuid)
+        failed = False
+        try:
+            uuid = self.resource.create(ndb=destination_kb, **data)
+        except exceptions.RateLimitError as e:
+            failed = True
+            delay = 5
+            if e.try_after:
+                delay = (e.try_after - datetime.utcnow()).seconds
```

Comment:
try_after is a timestamp. You should convert it to datetime or substract datetime.uctnow().timestamp()

Reply:
(same below)